mysql-test/suite/encryption/r/bug_ps_6922.result

```diff
@@ -0,0 +1,8 @@
+# Below queries would crash the server. Check that they work OK now.
+SET expand_fast_index_creation=1;
+ALTER TABLE mysql.db ORDER BY db DESC;
```

Comment:
Do we really need expand_fast_index_creation for this bug to appear?
And the table must be in mysql tablespace? Wouldn't a rebuild of table in General tablespace throw the same assert/error?

Reply:
Yes - the crash happens when expand fast index creation readds the indexes.
And yes - we spent a lot of time trying to reproduce this with custom tables, but without luck. Yura found a possible explanation for this, see on slack.

Reply:
but a normal table with General tablesapce is not tried afaiu..

Reply:
I tried it, it didn't crash.

Reply:
Please add a test for partition tables too.

Reply:
added